src/peft/tuners/lora/variants.py

```diff
@@ -285,3 +285,22 @@ class DoraConv3dVariant(_DoraConvNdVariant):
     def init(module: Conv3d, adapter_name: str) -> None:
         dora_layer = DoraConv3dLayer(fan_in_fan_out=False)
         _DoraConvNdVariant.init_convd_variant(module, adapter_name, dora_layer=dora_layer)
+
+
+class SineLoraLinearVariant(LoraVariant):
+    @staticmethod
+    def init(module: Linear, adapter_name:str) -> None:
```

Comment:
```suggestion
    def init(module: Linear, adapter_name:str, **kwargs) -> None:
```

With PR #2455 now merged, `init()` receives all the parameters that `update_layer` receives.

Reply:
hmmmm I did not use that and do you think that is ok?